k9mail/src/main/res/layout/message_crypto_info_dialog.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
done. the icons in the two FrameLayouts are actually all above each other in the same position, which is why they are called _1, _2, and _3